torch/_inductor/cudagraph_trees.py

```diff
@@ -960,11 +960,19 @@ def check_static_inputs_are_stable(self, new_inputs):
                 self.static_input_data_ptrs[i]
                 for i in self.non_managed_static_input_idxs
             ]
-            for t, data_ptr in zip(static_tensors, data_ptrs):
-                torch._check(
-                    t.data_ptr() == data_ptr,
-                    lambda: f"static input data pointer changed from {data_ptr} to {t.data_ptr()}",
-                )
+            error_msg = "static input data pointer changed.\n"
+            for i, (t, data_ptr) in enumerate(zip(static_tensors, data_ptrs)):
+                index = self.non_managed_static_input_idxs[i]
+                if t.data_ptr() != data_ptr:
+                    error_msg += (
+                        f"input name: {self.wrapped_function.placeholders[index]}. "
```

Comment:
Our other error msgs use the original source code. I think that is more useful to the user than our aot_autograd generated input name/stack trace. We could augment it with graph name if you think it is helpful.

What do you htink ?